src/support/bits.h

```diff
@@ -65,7 +65,9 @@ template<typename T> int CountTrailingZeroes(T v) {
 template<typename T> int CountLeadingZeroes(T v) {
   return CountLeadingZeroes(typename std::make_unsigned<T>::type(v));
 }
-template<typename T> bool IsPowerOf2(T v) { return v != 0 && PopCount(v) == 1; }
+template<typename T> bool IsPowerOf2(T v) {
+  return v != 0 && (v & (v - 1)) == 0;
+}
```

Comment:
Nice 👍 I like that this lets us delete code.

Reply:
usually `popcnt(x) == 1` (btw it include x != 0 case also) faster but only if we have native popcnt support. Alos LLVM & GCC smart enought to replace pattern above to popcnt(x) == 1 if it possible